README.md

```diff
@@ -93,6 +93,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [Simple Line Icons](https://github.com/vaso2/nuxt-simple-line-icons) - Simple Line Icons for Nuxt.
 - [nuxt-sass-resources-loader](https://github.com/anteriovieira/nuxt-sass-resources-loader) - SASS resources (e.g. variables, mixins etc.) module for NuxtJs.
 - [laravel-nuxt](https://github.com/skyrpex/laravel-nuxt-js) - Build a SPA with Laravel and Nuxt.
+- [nuxt-laravel](https://github.com/m2sd/nuxt-laravel) - Updated version of laravel-nuxt that exteds nuxt cli command.
```

Comment:
Added after `laravel-nuxt` because it is based on it / is compatible with it's composer companion extension.
`laravel-nuxt` only supports nuxt1 whereas `nuxt-laravel` requires nuxt as a peer dependency and should be modular enought to provide continued support